integration_tests/benchmarks/object-detection/benchmark_script.py

```diff
@@ -232,61 +173,226 @@ def run_detailed_pr_curve_evaluation(dset: Dataset, model: Model):
+                        dset=dataset, model=model
+                    )
+                    eval_detail = run_detailed_pr_curve_evaluation(
+                        dset=dataset, model=model
+                    )
+                except TimeoutError:
+                    raise TimeoutError(
+                        f"Evaluation timed out when processing {limit} datums."
+                    )
+
+                start = time()
+                client.delete_model(model.name, timeout=30)
+                pd_deletion_time = time() - start
+
+                start = time()
+                client.delete_dataset(dataset.name, timeout=30)
+                gt_deletion_time = time() - start
+
+                gt_results[gt_type].add(
+                    dtype=gt_type,
+                    ingestion=gt_ingest_time,
+                    finalization=gt_finalization_time,
+                    deletion=gt_deletion_time,
+                )
+                pd_results[pd_type].add(
+                    dtype=pd_type,
+                    ingestion=pd_ingest_time,
+                    finalization=pd_finalization_time,
+                    deletion=pd_deletion_time,
+                )
+                eval_results[gt_type][pd_type].add(
+                    gt_type=gt_type,
+                    pd_type=pd_type,
+                    eval_base=eval_base.meta,
+                    eval_base_pr=eval_pr.meta,
+                    eval_base_pr_detail=eval_detail.meta,
+                )
+
+    results = {
+        "dataset": [result.result() for result in gt_results.values()],
+        "model": [result.result() for result in pd_results.values()],
+        "evaluation": [
+            result.result()
+            for permuation in eval_results.values()
+            for result in permuation.values()
+        ],
+    }
+    write_results_to_file(write_path=write_path, result_dict=results)
 
 
 if __name__ == "__main__":
-    run_benchmarking_analysis()
+
+    gt_box_filename = "gt_objdet_coco_bbox.jsonl"
+    gt_polygon_filename = "gt_objdet_coco_polygon.jsonl"
+    gt_multipolygon_filename = "gt_objdet_coco_raster_multipolygon.jsonl"
+    gt_raster_filename = "gt_objdet_coco_raster_bitmask.jsonl"
+    pd_box_filename = "pd_objdet_yolo_bbox.jsonl"
+    pd_polygon_filename = "pd_objdet_yolo_polygon.jsonl"
+    pd_multipolygon_filename = "pd_objdet_yolo_multipolygon.jsonl"
+    pd_raster_filename = "pd_objdet_yolo_raster.jsonl"
+
+    groundtruths = {
+        "box": gt_box_filename,
+        "polygon": gt_polygon_filename,
+        "multipolygon": gt_multipolygon_filename,
+        "raster": gt_raster_filename,
+    }
+    predictions = {
+        "box": pd_box_filename,
+        "polygon": pd_polygon_filename,
+        "multipolygon": pd_multipolygon_filename,
+        "raster": pd_raster_filename,
+    }
+
+    run_benchmarking_analysis(
+        groundtruths=groundtruths,
+        predictions=predictions,
+        limits_to_test=[12],
```

Comment:
can we add multiple tests (e.g., testing 12 limits_to_test multiple times)? 

it seems like it'd be difficult to tell from the output which test was which in the event that you had a bunch of limits_to_test (e.g., `limits_to_test=[1, 5, 10, ...]`. can we add back some indicator near the top of the output that makes it clear how limits_to_test varies across runs? this would be similar to `number_of_datums` in the classification output

Reply:
I reduced it to a single trial to save on time since running [12, 12] takes 15 minutes with all the type combinations.

Reply:
The runs are also averaged together to reduce output length

Reply:
we probably don't want to average them together so we can test different datum quantities